libs/community/tests/integration_tests/retrievers/document_compressors/test_base.py

```diff
@@ -27,3 +28,25 @@ def test_document_compressor_pipeline() -> None:
     actual = pipeline_filter.compress_documents(docs, "Tell me about farm animals")
     assert len(actual) == 1
     assert actual[0].page_content in texts[:2]
+
+
+@pytest.mark.asyncio
```

Comment:
```suggestion
```

Reply:
if I delete this annotation, the test is not run